src/main/java/io/spine/helloworld/client/Client.java

```diff
@@ -88,6 +89,24 @@ private void printEvent(EventMessage e) {
         System.out.println(out);
     }
 
+    /**
+     * Cancels all current subscriptions, if any.
+     */
+    private void cancelSubscriptions() {
+        if (subscriptions != null) {
+            subscriptions.forEach(s -> client.subscriptions().cancel(s));
+            this.subscriptions = null;
+        }
+    }
+
+    /**
+     * Verifies if the client finished cancelling active subscriptions.
```

Comment:
Maybe not `Verifies` but `tests` or `examines`? We don't perform any active actions, right? I may be mixing up contexts, but usually `verify` does something and not just returns the state.

Reply:
Right.